compiler/rustc_query_impl/src/plumbing.rs

```diff
@@ -503,11 +503,11 @@ macro_rules! define_queries {
                 pub(super) fn $name<'tcx>(
```

Comment:
Should we attempt to use the same convention in this file?

Reply:
Probably.